py/client/pydeephaven/agg.py

```diff
@@ -0,0 +1,363 @@
+            default is None, only valid when used in Table agg_all_by operation
+
+    Returns:
+        an aggregation
+    """
+    agg_spec = _GrpcAggSpec(median=_GrpcAggSpec.AggSpecMedian(average_evenly_divided=True))
+    return _AggregationColumns(agg_spec=agg_spec, cols=cols)
+
+
+def pct(percentile: float, cols: Union[str, List[str]] = None) -> Aggregation:
+    """Create a Percentile aggregation.
+
+    Args:
+        percentile (float): the percentile used for calculation
+        cols (Union[str, List[str]]): the column(s) to aggregate on, can be renaming expressions, i.e. "new_col = col";
+            default is None, only valid when used in Table agg_all_by operation
+
+    Returns:
+        an aggregation
+    """
+    agg_spec = _GrpcAggSpec(percentile=_GrpcAggSpec.AggSpecPercentile(percentile=percentile,
+                                                                      average_evenly_divided=True))
+    return _AggregationColumns(agg_spec=agg_spec, cols=cols)
+
+
+def sorted_first(order_by: str, cols: Union[str, List[str]] = None) -> Aggregation:
+    """Create a SortedFirst aggregation.
+
+    Args:
+        order_by (str): the column to sort by
+        cols (Union[str, List[str]]): the column(s) to aggregate on, can be renaming expressions, i.e. "new_col = col";
+            default is None, only valid when used in Table agg_all_by operation
+
+    Returns:
+        an aggregation
+    """
+    sorted_column = _GrpcAggSpec.AggSpecSortedColumn(column_name=order_by)
+    agg_spec = _GrpcAggSpec(sorted_first=_GrpcAggSpec.AggSpecSorted(columns=[sorted_column]))
+    return _AggregationColumns(agg_spec=agg_spec, cols=cols)
+
+
+def sorted_last(order_by: str, cols: Union[str, List[str]] = None) -> Aggregation:
+    """Create a SortedLast aggregation.
+
+    Args:
+        order_by (str): the column to sort by
+        cols (Union[str, List[str]]): the column(s) to aggregate on, can be renaming expressions, i.e. "new_col = col";
+            default is None, only valid when used in Table agg_all_by operation
+
+    Returns:
+        an aggregation
+    """
+    sorted_column = _GrpcAggSpec.AggSpecSortedColumn(column_name=order_by)
+    agg_spec = _GrpcAggSpec(sorted_last=_GrpcAggSpec.AggSpecSorted(columns=[sorted_column]))
+    return _AggregationColumns(agg_spec=agg_spec, cols=cols)
+
+
+def std(cols: Union[str, List[str]] = None) -> Aggregation:
+    """Create a Std aggregation.
+
+    Args:
+        cols (Union[str, List[str]]): the column(s) to aggregate on, can be renaming expressions, i.e. "new_col = col";
+            default is None, only valid when used in Table agg_all_by operation
+
+    Returns:
+        an aggregation
+    """
+    agg_spec = _GrpcAggSpec(std=_GrpcAggSpec.AggSpecStd())
+    return _AggregationColumns(agg_spec=agg_spec, cols=cols)
+
+
+def unique(cols: Union[str, List[str]] = None) -> Aggregation:
+    """Create a Unique aggregation.
```

Comment:
Many of these pydocs are not clear to users because they refer to something that the user will not know.  How is the noob supposed to interpret "Unique aggregation"?

Reply:
filed (https://github.com/deephaven/deephaven-core/issues/3624)